HTTPServer.driver/Driver.cs

```diff
@@ -32,6 +32,7 @@ public static void Main(string[] args)
             commandDetails.Add(Tuple.Create((ICriteria)new VersionNotSupportedCriteria(), (IHttpHandler)new VersionNotSupported()));
             commandDetails.Add(Tuple.Create((ICriteria)new ContentsCriteria(), (IHttpHandler)new GetContents(pathContents)));
             commandDetails.Add(Tuple.Create((ICriteria)new PostCriteria(), (IHttpHandler)new PostContents(pathContents)));
+            commandDetails.Add(Tuple.Create((ICriteria)new PutCriteria(), (IHttpHandler)new PutContents(pathContents)));
```

Comment:
yeah, it's expected that the application code would have specific details of what needs to actually happen for a given request so it can configure the server that way. it's just that that layer shouldn't be concerned with details like exactly how the router stores the data